### PR TITLE
Fix Bug 1245208, add plugincheck layout variations for newsletter signup testing

### PR DESCRIPTION
**Pull request by schalkneethling**
@craigcook r?

### CODE REVIEW

**Comment by schalkneethling** [comment]
Should have added this is the description of the PR. The pages are:

## Current
/plugincheck/

## Variants

/plugincheck/?v=b
/plugincheck/?v=c
/plugincheck/?v=d

File: media/css/plugincheck/plugincheck.less [hunk 1]
```diff
@@ -381,10 +387,215 @@ html[dir="rtl"] {
...
+
+        .firefox {
+            font-weight: bold;
+        }
+    }
+
+    h4 {
+        .font-size(2rem);
+        line-height: 1.2;
+        letter-spacing: 0;
```
Comment (craigcook) [inline media/css/plugincheck/plugincheck.less]: I think this should be `letter-spacing: normal;` though the effect looks to be the same (probably depends on fonts).

**Comment by craigcook** [comment]
Looking good, though the positioning of the logo in the header feels wrong at desktop and tablet sizes: 

plugincheck-header

I think it should be more aligned with the headline. We could put the logo inside the `main-feature` block to make it a bit easier, or else keep it where it is and use some padding and negative margin trickery to move things around like this (hacked in devtools):

plugincheck-header-2

File: media/css/plugincheck/plugincheck.less (hunk 1 above)
Comment (schalkneethling) [inline media/css/plugincheck/plugincheck.less]: `normal` sounds more appropriate, will change.

**Comment by schalkneethling** [comment]
Hmmm, yeah. I followed the design but, you are correct so, not sure. Should I just go ahead and change it or, first fly this by Michael?

**Comment by craigcook** [comment]
Yeah, let's run it by Michael and see what he thinks.

**Comment by designerham** [comment]
@schalkneethling and @craigcook I prefer the second version where the logo is in line with the header an sub head – much nicer. Thanks!

**Comment by schalkneethling** [comment]
@craigcook updated, and ready to go. Thx

**Comment by craigcook** [comment]
r+ :lollipop:

**Pull request merged**